test/draw/RasterRgbTest.ooc

```diff
@@ -78,45 +78,36 @@ RasterRgbTest: class extends Fixture {
 			(image, image2) referenceCount decrease()
 			outputFast free()
 		})
-		this add("coordinate systems", func {
+		this add("flipping", func {
 			output := "test/draw/output/coordinateswap.png"
 			image := RasterRgb open(this sourceSpace)
 			image2 := image copy()
 
 			image referenceCount decrease()
-			image = RasterRgb open(this sourceSpace, CoordinateSystem YUpward)
-			DrawState new(image2) setInputImage(image) setInterpolate(false) draw()
+			image = RasterRgb open(this sourceSpace)
+			DrawState new(image2) setInputImage(image) setFlipSourceY(true) setInterpolate(false) draw()
 			for (row in 0 .. image2 height)
-				for (column in 0 .. image2 width)
+			for (column in 0 .. image2 width)
 					expect(image2[column, row] == image[column, image height - row - 1])
 
 			image referenceCount decrease()
 			image = RasterRgb open(this sourceSpace)
 			image2 save(output) . free()
-			image2 = RasterRgb open(output, CoordinateSystem XLeftward | CoordinateSystem YUpward)
-			DrawState new(image2) setInputImage(image) setInterpolate(false) draw()
+			image2 = RasterRgb open(output)
+			DrawState new(image2) setInputImage(image) setFlipSourceX(true) setFlipSourceY(true) setInterpolate(false) draw()
 			for (row in 0 .. image2 height)
 				for (column in 0 .. image2 width)
 					expect(image2[column, row] == image[image width - column - 1, image height - row - 1])
 
 			image referenceCount decrease()
-			image = RasterRgb open(this sourceSpace, CoordinateSystem XLeftward)
+			image = RasterRgb open(this sourceSpace)
 			image2 save(output) . free()
 			image2 = RasterRgb open(output)
-			DrawState new(image2) setInputImage(image) setInterpolate(false) draw()
+			DrawState new(image2) setInputImage(image) setFlipSourceX(true) setInterpolate(false) draw()
 			for (row in 0 .. image2 height)
 				for (column in 0 .. image2 width)
 					expect(image2[column, row] == image[image width - column - 1, row])
 
-			image referenceCount decrease()
-			image = RasterRgb open(this sourceSpace)
-			image2 save(output) . free()
-			image2 = RasterRgb open(output, CoordinateSystem XRightward | CoordinateSystem YDownward)
-			DrawState new(image2) setInputImage(image) setInterpolate(false) draw()
-			for (row in 0 .. image2 height)
-				for (column in 0 .. image2 width)
-					expect(image2[column, row] == image[column, row])
-
```

Comment:
Translating this test would make no sense since I now describe the change in systems instead of the actual coordinate systems.